docs/site/Application-generator.md

```diff
@@ -103,3 +103,65 @@ greeted with a message.
 Once the application has been created, additional generators such as
 [controller generator](Controller-generator.md) can be run from the
 application's root directory to further scaffold the application.
+
+
+### Dockerfile Considerations
```

Comment:
Can you please add a link to this section to the top, where we are describing `--docker` option? Something along the following lines:

```markdown
`--docker`: Generate Dockerfile and add npm scripts to build/run the project in a docker container. 
See also [Dockerfile considerations](#dockerfile-consideration) below.